Bruin Bite/Supporting Files/LoginAlertPresentable.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LoginAlertPresentable.swift
+//  Bruin Bite
+//
+//  Created by Samuel J. Lee on 4/22/19.
+//  Copyright © 2019 Dont Eat Alone. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// To use this protocol, make a segue with identifier "login"
+
+protocol LoginAlertPresentable: class {}
+
+extension LoginAlertPresentable where Self: UIViewController {
+    func presentAlert() {
```

Comment:
definitely rename this function. Maybe something like `presentNotLoggedInAlert()`?

Reply:
thanks!